wzdx/raw_to_standard/icone.py

```diff
@@ -1,58 +1,96 @@
 import argparse
+import datetime
 import json
 import logging
 import time
 import uuid
 import xml.etree.ElementTree as ET
 from collections import OrderedDict
 
-from wzdx.tools import date_tools, polygon_tools, wzdx_translator
-from wzdx.util.collections import PathDict
+from ..tools import date_tools, geospatial_tools, wzdx_translator, combination
+from ..util.collections import PathDict
 
-PROGRAM_NAME = 'iConeRawToStandard'
-PROGRAM_VERSION = '1.0'
+PROGRAM_NAME = "iConeRawToStandard"
+PROGRAM_VERSION = "1.0"
 
 
 def main():
     input_file, output_dir = parse_rtdh_arguments()
-    input_file_contents = open(input_file, 'r').read()
-    generated_messages = generate_standard_messages_from_string(
-        input_file_contents)
+    input_file_contents = open(input_file, "r").read()
+    generated_messages = generate_standard_messages_from_string(input_file_contents)
 
     generated_files_list = []
+    features = json.loads(open(f"{output_dir}/icone_feature_collection.geojson").read())
+
+    wzdx_msg = {}
     for message in generated_messages:
-        output_path = f"{output_dir}/standard_icone_{message['event']['source']['id']}_{round(message['rtdh_timestamp'])}.json"
-        open(output_path, 'w+').write(json.dumps(message, indent=2))
+        output_path = f"{output_dir}/icone_{message['event']['source']['id']}_{round(message['rtdh_timestamp'])}_{message['event']['detail']['direction']}.json"
+        open(output_path, "w+").write(json.dumps(message, indent=2))
         generated_files_list.append(output_path)
 
+        features.append(
+            {
+                "type": "Feature",
+                "properties": {
+                    "id": message["event"]["source"]["id"],
+                    "route_details": message["event"]["additional_info"][
+                        "route_details_start"
+                    ],
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": message["event"]["geometry"][0],
+                },
+            }
+        )
+
+    open(f"{output_dir}/icone_feature_collection.geojson", "w+").write(
+        json.dumps(features, indent=2)
+    )
+
     if generated_files_list:
-        print(
-            f"Successfully generated standard message files: {generated_files_list}")
+        print(f"Successfully generated standard message files: {generated_files_list}")
```

Comment:
Let's swap the prints out for our logging statements

Reply:
Updated other logging statements to use print